assets/css/detours/_detour_map.scss

```diff
@@ -12,6 +14,12 @@
   }
 }
 
+.c-detour_map--original-route-shape-diverted {
+  stroke: semantic.$missed-stop;
```

Comment:
@mbta/skate-developers just want to sanity-check that I'm doing this in the way that makes the most sense. I just sort of flailed around until I was able to reference the `missed-stop` color in a way that worked, but maybe there's a better way?

Reply:
Looks correct to me